.evergreen/compass_package.sh

```diff
@@ -2,6 +2,14 @@
 
 set -e
 
+echo "WINDOWS_SIGNING_SERVER_HOSTNAME: ${WINDOWS_SIGNING_SERVER_HOSTNAME}"
+echo "WINDOWS_SIGNING_SERVER_PRIVATE_KEY: ${WINDOWS_SIGNING_SERVER_PRIVATE_KEY}"
```

Comment:
This is a private key identifier and not the actual contents of the private key, right?

Reply:
Correct, its just the absolute path to the file.